### ISSUE TITLE
Hyper-V hosts cannot connect to NC

### ISSUE THREAD

**Issue by playground-io** [microsoft/SDN#607]
Hi there,
We have implemented a LAB with ASHCI 23h2 (latest version) that run just fine no errors. After that we use the latest version of SDNExpress to implement SDN network, only the NC (simplest case) no SLB or GW.
Everything seems to be ok, but the configurations applied in Admin Center, seems to be applied correctly, but they are not being pushed to the Hyper-V hosts.
In the beginning all communications in port 6640 were TIME_WAIT, then we discovered that on the latest 23h2 the port 6645 it was also used to communicate with the hosts and they were also in TIME_WAIT. Then we deleted all the certificates added by Azure in hyper-v hosts, leaving only the self-sign, after this all connections have passed to Established. A couple minutes later besides the connections in port 6640 and 6645, maintains Established, some routine put back the Azure certificates into the hosts. Even with the connections in Established status, all configurations are accepted correctly in the Admin Center, seems not to be pushed to the Hyper-V hosts.
The SDNDiagnostic says that all services in NC are ok, so it seems the problem is just a matter of communication between NC and Hyper-V nodes.
Have someone tried to install SDN with the ASHCI 23h2?
Thanks

**Comment by The-TeQnician** [comment]
Azure Stack HCI 23H2 and the resource bridge are not supported/compatible yet with SDN. So i guess very few have tried this. Are you running virtual or physical lab environment?

**Comment by AnirbanPaul** [comment]
SDN is fully supported on Azure Stack HCI 23H2. While the installation is based off existing tools (SDN Express or Windows Admin Center), full integration with the Azure portal is coming soon.
@playground-io, can you please send email to sdn_feedback@microsoft.com detailing the issue.

**Comment by playground-io** [comment]
I'll do it.

Thanks

**Comment by playground-io** [comment]
> SDN is fully supported on Azure Stack HCI 23H2. While the installation is based off existing tools (SDN Express or Windows Admin Center), full integration with the Azure portal is coming soon. @playground-io, can you please send email to [sdn_feedback@microsoft.com](mailto:sdn_feedback@microsoft.com) detailing the issue.

Is there some place with the new manual installation procedures?

**Comment by playground-io** [comment]
In order to have a clean environment to troubleshoot, I've reinstalled the lab from scratch.
After all the installation everything maintained the same behaviour, not working properly. Then I decided to upgrade both Operating Systems in the NC (through Sconfig) and in the Hyper-V hosts (through Azure Portal). After this, still no luck but I noted that the Azure Upgrade process has duplicated all the certificates in the cert:\localmachine\my for all hyper-v hosts. I deleted some of the old certificates that were duplicated and after that, the communication between the NC and the Host started to work as expected.
